aspnetcore/blazor/performance/profiling.md

```diff
@@ -0,0 +1,226 @@
+---
+title: ASP.NET Core Blazor WebAssembly performance profiling and diagnostic counters
+author: guardrex
+description: Learn about performance profiling and diagnostic counters in ASP.NET Core Blazor WebAssembly apps.
+monikerRange: '>= aspnetcore-10.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/16/2025
+uid: blazor/performance/profiling
+---
+# ASP.NET Core Blazor WebAssembly performance profiling and diagnostic counters
+
+<!-- UPDATE 10.0 - Activate ...
+
+[!INCLUDE[](~/includes/not-latest-version.md)]
+
+-->
+
+This article describes performance profiling tools and diagnostic counters for Blazor WebAssembly apps.
+
+## Prerequisite
+
+Install the [.NET WebAssembly build tools](xref:blazor/tooling/webassembly#net-webassembly-build-tools):
+
+```dotnetcli
+dotnet workload install wasm-tools
+```
+
+## Browser developer tools
+
+App code can be manually profiled using the performance profiler in a browser's developer tools console.
+
+Built-in performance counters are available to track:
+
+* [Ahead-of-time (AOT) compilation](xref:blazor/tooling/webassembly#ahead-of-time-aot-compilation)
+* Code interpolation
+* [JIT (Just-In-Time) interpolation](https://developer.mozilla.org/docs/Glossary/Just_In_Time_Compilation) 
```

Comment:
We don't do any JIT for WASM, instead we do `jiterp` but it's too low level detail. Please drop the JIT mention.